tests/read_from_stdin.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * read_from_stdin.cpp
+ *
+ *  Created on: Oct 19, 2018
+ *      Author: martinelli
+ */
+
+#include <iostream>
+#include <string>
+
+#include <pico/pico.hpp>
+
+#include <catch.hpp>
+
+#include "common/io.hpp"
+
+using namespace pico;
```

Comment:
(minor issue) golden rule: avoid global `using namespace` directives (we need to remove them throughout the whole repository as well)